### DISCUSSION TITLE
Using tomography data with OpenPNM

### DISCUSSION THREAD

**Discussion by fsamaraie** [PMEAL/OpenPNM#2694] [Q&A] [answered]
Hello Community!
I am new to openpnm and I would appreciate your help.
I have segmented (binary) XCT data but I don't know how to extract their pore network data. In openpnm tutorials, I can only see that the pore network are generated using other functions rather than being imported. 
I would appreciate any advice or guidance. 
Thanks

**Comment by ma-sadeghi** [comment, marked as answer]
Hi. We have a few examples in our other repository, PoreSpy. See [here](https://porespy.org/examples/networks/tutorials/snow_basic.html) for instance.

**Reply by fsamaraie** [reply]
Many thanks
